web/components/bet/sell-row.tsx

```diff
@@ -131,3 +138,128 @@ function SellSharesModal(props: {
     </Modal>
   )
 }
+
+export function SellStonkRow(props: {
```

Comment:
They worked sufficiently differently by displaying the sell value of the shares instead of the value of the shares at resolution. But bc that should all be changed over at once I'll revert those changes.